dawn/renderer/components/Editor.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import ConsoleOutput from './ConsoleOutput';
-import Ansible from '../utils/Ansible';
+// import Ansible from '../../main/ansible/Ansible';
```

Comment:
I think you should just delete the line completely, rather than commenting it out. Committing commented out lines leads to clutter. If we need to go back, we can just use git to revert.

Same for all the commented out stuff below.